<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE html SYSTEM "about:legacy-compat">
<html xmlns="http://www.w3.org/1999/xhtml" lang="en">
  <head>
    <title>phpDox - PHPUnit_Framework_TestFailure</title>
    <link rel="stylesheet" type="text/css" href="../css/style.css" media="screen"/>
    <meta http-equiv="content-type" content="text/html; charset=utf-8"/>
  </head>
  <body>
    <nav class="topnav">
      <ul>
        <li>
          <div class="logo"><span>/**</span>phpDox</div>
        </li>
        <li class="separator">
          <a href="../index.xhtml">Overview</a>
        </li>
        <li class="separator">
          <a href="../namespaces.xhtml">Namespaces</a>
        </li>
        <li>
          <a href="../interfaces.xhtml">Interfaces</a>
        </li>
        <li>
          <a href="../classes.xhtml">Classes</a>
        </li>
        <li>
          <a href="../traits.xhtml">Traits</a>
        </li>
        <li class="separator">
          <a href="../source/index.xhtml">Source</a>
        </li>
      </ul>
    </nav>
    <div id="mainstage">
      <div class="box">
        <ul class="breadcrumb">
          <li>
            <a href="../index.xhtml">Overview</a>
          </li>
          <li class="separator">
            <a href="../classes.xhtml">Classes</a>
          </li>
          <li class="separator">PHPUnit_Framework_TestFailure</li>
        </ul>
      </div>
      <nav class="box">
        <ul>
          <li>
            <a href="#introduction">Introduction</a>
          </li>
          <li>
            <a href="#synopsis">Synopsis</a>
          </li>
          <li>
            <a href="#members">Members</a>
          </li>
          <li>
            <a href="#methods">Methods</a>
          </li>
          <li>
            <a href="../source/vendor/phpunit/phpunit/src/Framework/TestFailure.php.xhtml#line16">Source</a>
          </li>
        </ul>
      </nav>
      <section>
        <h1 id="introduction"><small>\</small>PHPUnit_Framework_TestFailure</h1>
        <h4>A TestFailure collects a failed test together with the caught exception.</h4>
        <p/>
        <ul/>
        <h2 id="synopsis">Synopsis</h2>
        <div class="synopsis">class PHPUnit_Framework_TestFailure
            {<br/><ul class="none"><li>// members</li><li>private  <a href="#members">$testName</a>;
                    </li><li>protected <span title="PHPUnit_Framework_Test|null">PHPUnit_Framework_Test|null</span> <a href="#members">$failedTest</a>;
                    </li><li>protected <span title="Exception">Exception</span> <a href="#members">$thrownException</a>;
                    </li></ul><ul class="none"><li>// methods</li><li>public void <a title="PHPUnit_Framework_TestFailure" href="../classes/PHPUnit_Framework_TestFailure/__construct.xhtml">__construct</a>()
                    </li><li>public string <a title="PHPUnit_Framework_TestFailure" href="../classes/PHPUnit_Framework_TestFailure/toString.xhtml">toString</a>()
                    </li><li>public string <a title="PHPUnit_Framework_TestFailure" href="../classes/PHPUnit_Framework_TestFailure/getExceptionAsString.xhtml">getExceptionAsString</a>()
                    </li><li>public static string <a title="PHPUnit_Framework_TestFailure" href="../classes/PHPUnit_Framework_TestFailure/exceptionToString.xhtml">exceptionToString</a>()
                    </li><li>public string <a title="PHPUnit_Framework_TestFailure" href="../classes/PHPUnit_Framework_TestFailure/getTestName.xhtml">getTestName</a>()
                    </li><li>public PHPUnit_Framework_Test|null <a title="PHPUnit_Framework_TestFailure" href="../classes/PHPUnit_Framework_TestFailure/failedTest.xhtml">failedTest</a>()
                    </li><li>public Exception <a title="PHPUnit_Framework_TestFailure" href="../classes/PHPUnit_Framework_TestFailure/thrownException.xhtml">thrownException</a>()
                    </li><li>public string <a title="PHPUnit_Framework_TestFailure" href="../classes/PHPUnit_Framework_TestFailure/exceptionMessage.xhtml">exceptionMessage</a>()
                    </li><li>public bool <a title="PHPUnit_Framework_TestFailure" href="../classes/PHPUnit_Framework_TestFailure/isFailure.xhtml">isFailure</a>()
                    </li></ul>

            }<br/></div>
        <h2 id="members">Members</h2>
        <div class="styled members">
          <h4>private</h4>
          <ul class="members">
            <li id="testName"><strong>$testName</strong>
                —
                string</li>
          </ul>
          <h4>protected</h4>
          <ul class="members">
            <li id="failedTest"><strong>$failedTest</strong>
                —
                <a title="PHPUnit_Framework_Test|null" href="../classes/PHPUnit_Framework_Test|null.xhtml">PHPUnit_Framework_Test|null</a></li>
            <li id="thrownException"><strong>$thrownException</strong>
                —
                <a title="Exception" href="../classes/Exception.xhtml">Exception</a></li>
          </ul>
        </div>
        <h2 id="methods">Methods</h2>
        <div class="styled">
          <h4>public</h4>
          <ul>
            <li id="__construct"><a title="PHPUnit_Framework_TestFailure" href="../classes/PHPUnit_Framework_TestFailure/__construct.xhtml">__construct()</a>
                — Constructs a TestFailure with the given test and exception.</li>
            <li id="exceptionMessage"><a title="PHPUnit_Framework_TestFailure" href="../classes/PHPUnit_Framework_TestFailure/exceptionMessage.xhtml">exceptionMessage()</a>
                — Returns the exception's message.</li>
            <li id="exceptionToString"><a title="PHPUnit_Framework_TestFailure" href="../classes/PHPUnit_Framework_TestFailure/exceptionToString.xhtml">exceptionToString()</a>
                — Returns a description for an exception.</li>
            <li id="failedTest"><a title="PHPUnit_Framework_TestFailure" href="../classes/PHPUnit_Framework_TestFailure/failedTest.xhtml">failedTest()</a>
                — Returns the failing test.</li>
            <li id="getExceptionAsString"><a title="PHPUnit_Framework_TestFailure" href="../classes/PHPUnit_Framework_TestFailure/getExceptionAsString.xhtml">getExceptionAsString()</a>
                — Returns a description for the thrown exception.</li>
            <li id="getTestName"><a title="PHPUnit_Framework_TestFailure" href="../classes/PHPUnit_Framework_TestFailure/getTestName.xhtml">getTestName()</a>
                — Returns the name of the failing test (including data set, if any).</li>
            <li id="isFailure"><a title="PHPUnit_Framework_TestFailure" href="../classes/PHPUnit_Framework_TestFailure/isFailure.xhtml">isFailure()</a>
                — Returns true if the thrown exception is of type AssertionFailedError.</li>
            <li id="thrownException"><a title="PHPUnit_Framework_TestFailure" href="../classes/PHPUnit_Framework_TestFailure/thrownException.xhtml">thrownException()</a>
                — Gets the thrown exception.</li>
            <li id="toString"><a title="PHPUnit_Framework_TestFailure" href="../classes/PHPUnit_Framework_TestFailure/toString.xhtml">toString()</a>
                — Returns a short description of the failure.</li>
          </ul>
        </div>
      </section>
    </div>
    <footer>
      <span>Generated using phpDox 0.8.0 - Copyright (C) 2010 - 2016 by Arne Blankerts</span>
    </footer>
  </body>
</html>
